app/models/user/permissions.rb

```diff
@@ -99,6 +99,10 @@ def forbidden_some_services?
     !has_access_to_all_services? && account.provider_can_use?(:service_permissions)
   end
 
+  def access_to_service_admin_sections?
+    (member_permission_ids & %i[partners plans monitoring]).any? && accessible_services?
```

Comment:
As mention in https://github.com/3scale/porta/pull/2244/files#r519730489, these 'service admin sections' should be declared in a constant there.